src/main/java/org/folio/domain/ActionBlocks.java

```diff
@@ -92,14 +108,17 @@ else if (condition == RECALL_OVERDUE_BY_MAX_NUMBER_OF_DAYS) {
         .anyMatch(days -> days > limitValue);
     }
     else if (condition == MAX_OUTSTANDING_FEE_FINE_BALANCE) {
+      log.info(LOG_TEMPLATE_BY_LIMIT_CONDITION, condition);
```

Comment:
maybe it makes sense to log a condition once and log the value of blocks for each condition